From 962194fe25c390c5859bc525e53adb1938ee59e6 Mon Sep 17 00:00:00 2001
From: Megha Dey <megha.dey@linux.intel.com>
Date: Tue, 19 Apr 2016 12:06:22 -0700
Subject: [PATCH 1/2] ndctl: fix build on systems where /bin/sh is not a link
 to bash

On Debian and other Debian-based Linux distributions, autogen.sh
with the following error:
    ./git-version: 37: ./git-version: Bad substitution

This happens that tye git-version script is using a 'bashim' to
perform a substring extraction. Thus, this can only be performed
using bash. Hence shifting from sh to bash.

Signed-off-by: Megha Dey <megha.dey@linux.intel.com>
---
 git-version-gen | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/git-version-gen b/git-version-gen
index c78aff4..629f6d4 100755
--- a/git-version-gen
+++ b/git-version-gen
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 dirty() {
 	git update-index -q --refresh
-- 
1.9.1

